api/filter-alerts.go

```diff
@@ -0,0 +1,235 @@
+package api
+
+import (
+	"fmt"
+	graphql "github.com/cli/shurcooL-graphql"
+	"github.com/humio/cli/api/internal/humiographql"
+)
+
+type FilterAlert struct {
+	ID                 string   `graphql:"id"             yaml:"-"                       json:"id"`
+	Name               string   `graphql:"name"           yaml:"name"                    json:"name"`
+	Description        string   `graphql:"description"    yaml:"description,omitempty"   json:"description,omitempty"`
+	QueryString        string   `graphql:"queryString"    yaml:"queryString"             json:"queryString"`
+	Actions            []string `graphql:"actions"        yaml:"actions"                 json:"actions"`
```

Comment:
Rename to `ActionNames`?

Reply:
I thought of that, but actionIDs as input will actually work 😓 

actionID/actionName -> api -> graphql -> api -> actionName

Change it? Yay/Nay

Reply:
Renamed to ActionNames